config.go

```diff
@@ -0,0 +1,53 @@
+/**
+ *  Copyright (c) 2021  Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package candlelight
+
+import "go.opentelemetry.io/otel/trace"
+
+// Config object for otel tracing
```

Comment:
This isn't a sentence.  It would also be nice to provide a bit more information.

Reply:
Modified comments as suggested.